### DISCUSSION TITLE
Use GitHub organisation templates

### DISCUSSION THREAD

**Discussion by markusweigelt** [kitodo/kitodo-production#4883] [Q&A]
Clearer issue description and so fewer queries to the issue creator and communication overhead.

For example:
https://github.com/pods-framework/pods/issues/new/choose

**Comment by solth** [comment]
We do have a few issue templates for bug reports and feature requests now. Do we need more or does this cover most cases? I personally think we should limit the number of templates to a low number to not make the selection too confusing, but if there is need for one or two more templates that would still be well within such a limit.
